packages/uui-ref-node/lib/uui-ref-node.element.ts

```diff
@@ -32,6 +37,17 @@ export class UUINodeListItemElement extends UUIBaseListItemElement {
         margin-right: var(--uui-size-1);
       }
 
+      #icon-fallback {
+        display: inline-block;
+        vertical-align: bottom;
+        width: 1.15em;
+        height: 1.15em;
+      }
+
+      #icon-fallback svg {
+        fill: currentColor;
+      }
```

Comment:
I've just added this, to get the same feature as uui-icon would have provided.

I'm not sure its the right choice to avoid using uui-icon here. I´m affrighted we will lose expected behaviour when having fallback and that we might not notice such. So how bad would it be touse uui-icon for fallback in your perspective?

Otherwise looks good and I´m good to merge when we have decided on that. :-)

Thanks

Reply:
I have updated the PR to use uui-icon instead of the custom icon implementation. Please have an extra look and feel free to merge when the checks are all green.